README.md

```diff
@@ -2,6 +2,17 @@
 
 This adds [Julia language](https://julialang.org/) support to [CTAGS](https://ctags.io/).
 
+To install run
+```julialang
+using Pkg
+Pkg.add("Interpolations")
+```
+or
+```
+] add JuliaCtags
+```
+To obtain the ctags configuration file use `get_config()` which will create a file called `julia_ctags` in the current directory.
+
```

Comment:
Need to split this section from the below,
and maybe briefly say how the created file is used in the below process.